src/wast-parser.y

```diff
@@ -1531,14 +1500,14 @@ script :
 
       int last_module_index = -1;
       for (size_t i = 0; i < $$->commands.size(); ++i) {
-        Command& command = *$$->commands[i].get();
+        Command* command = $$->commands[i].get();
```

Comment:
Somehow, this is screaming for an abstraction on the command object. However, I wouldn't worry about changing that on this PR.